be/src/runtime/statistic_result_writer.cpp

```diff
@@ -281,6 +285,61 @@ Status StatisticResultWriter::_fill_table_statistic_data(int version, const Colu
     return Status::OK();
 }
 
+/*
+FE SQL:    
+    SELECT cast(4 as INT), 
+         cast(7164015 as BIGINT), 
+         'k1', 
+         cast(COUNT(1) as BIGINT), 
+         cast(COUNT(1) * 4 as BIGINT), 
+         hll_serialize(IFNULL(hll_raw(`k1`), hll_empty())), 
```

Comment:
why hll_raw still need hll_serialize?

Reply:
we need send the hll's data to fe, soo need serialize function

Reply:
why could we directly send the hll data to FE? which should work.